### ISSUE TITLE
Flow check fails with 'Unexpected end of input' when project uses webpack and babel

### ISSUE THREAD

**Issue by richardsabow** [facebook/flow#353]
I tried to use `flow` combined with `webpack` and `babel` (for more `ES6` features), and i ran in to this error:
```
<path_to_project>/node_modules/webpack/node_modules/enhanced-resolve/test/fixtures/node_modules/invalidPackageJson/package.json:1:3,2: Unexpected end of input
```
Not sure if it's a `flow` issue, but any idea?

**Comment by iammerrick** [comment]
+1

**Comment by samwgoldman** [comment]
`node_modules/invalidPackageJson/package.json`...

Definitely a flow issue, but a funny one!

**Comment by ameyms** [comment]
+1

**Comment by rainboxx** [comment]
The node module `npmconf` also provides an empty `package.json` as fixture and `flow` crashed due to this. Unfortunately this renders `flow` unusable when having such a file in the project (even under `node_modules`). I know you can ignore specific files and folders but this is not a nice way to deal with this issue ;).

**Comment by samwgoldman** [comment]
Actually, when this issue was opened, the problem that will be fixed in the next release wasn't a problem yet—you actually got a useful parse error.

You can fix this today by adding an entry to the `[ignore]` section of the `.flowconfig` to exclude `enhanced-resolve/test`, which should never be needed in the first place.

Since there's a workaround, I'm going to go ahead and close this now. Please re-open if you are still having difficulties.

**Comment by samwgoldman** [comment]
Tracking the long-term fix in #561.
